exporter/awsemfexporter/doc.go

```diff
@@ -14,4 +14,6 @@
 
 // Package awsemfexporter implements an OpenTelemetry Collector exporter that sends EmbeddedMetricFormat to
 // AWS CloudWatch Logs in the region the collector is running in using the PutLogEvents API.
+//
+//go:generate mdatagen metadata.yaml
```

Comment:
for consistency w/ the other packages, please leave the docstring above the package definition